### ISSUE TITLE
Custom Queries

### ISSUE THREAD

**Issue by zn2t** [smartmarmot/DBforBIX#39]
Hello, Smartmarmot!

First, i want to thank you for this awesome piece of software you made. I do use Orabbix for few years, and it is just awesome. Now I'm trying to monitor some MySQL instances I have with the DBforBIX, and some doubt comes in mind. like:

Can we create some custom queries like we could do in the old query.props way?
I have tried to add a line at the "config.properties" file, something like:

DB.MYSQLDB2.QueryListFile=./query.props

I also created the "query.props" file and filed it with some query (like we do in the old Orabbix configuration way), I created the item in Zabbix, but I couldn't make it work.

Also, I tried to create an item type "Database monitor" in Zabbix with a simple query. and again, couldn't make it work. 

Any idea how to make such thing work? or can you add this feature in the future?

Best Regards!

**Comment by KJlt77** [comment]
Hi, zn2t.

There were a lot of changes in 2.4 version. You should read all 3 wiki pages here on github first.

Best Regards.

**Comment by smartmarmot** [comment]
Since I've done a lot of changes with the new version of DBforBIX 3.0 I've decided to create a branch that contains the old 2.X version. you can find it here: https://github.com/smartmarmot/DBforBIX/tree/DBforBIX_2.X

**Comment by zn2t** [comment]
Hi, KJLt77 and Smartmarmot.

Thank you, both for the response. After your comments, I found that I was trying to do it the old way.
Also digging the wiki (as mentioned by KJLt77) I found the "[Some Explanation](https://github.com/smartmarmot/DBforBIX/wiki/New-DBforBIX-as-Zabbix-Proxy#some-explanation)" section, with shows me the direction to create custom queries in the new DBforBIX.

As a request, maybe, we can have a new page in the Wiki, dedicated to the creation of custom queries, with images/samples. Like me, there may be other in the same doubt.

Again, thank you for this awesome tool.

Best Regards!
